### PR TITLE
Changed HarmEffect

### PR DESCRIPTION
**Pull request by camelliaguan3**

### CODE REVIEW

File: src/main/java/org/terasology/potions/effect/HarmEffect.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     * The base amount of harming allowed. It's intended to be later multiplied by the magnitude to get the final
+     * harming amount.
+     */
+    private int baseDamage = 10;
+
+    /**
+     * Apply the harming effect onto the entity. This will harm 10 (or baseDamage) * magnitude HP.
+     *
+     * @param instigator    The instigator who is applying this effect on the entity. It can be a herb, potion, etc.
+     * @param entity        The entity who the healing effect is being applied on.
```
Comment (xrtariq2594) [inline src/main/java/org/terasology/potions/effect/HarmEffect.java]: entity who the _harming_ effect.

**Comment by xrtariq2594** [comment]
Just that fix remains, and this will be ready for merging :).

**Comment by camelliaguan3** [comment]
I fixed it

**Comment by xrtariq2594** [comment]
Merged :)

**Pull request merged**